tools/who_what_benchmark/tests/test_cli_reranking.py

```diff
@@ -13,40 +13,36 @@
 tmp_dir = tempfile.mkdtemp()
 
 
-OV_RERANK_MODELS = {
-    ("cross-encoder/ms-marco-TinyBERT-L2-v2", "text-classification"),
-    ("Qwen/Qwen3-Reranker-0.6B", "text-generation"),
-}
+def download_model(model_id, task, tmp_path):
+    MODEL_PATH = Path(tmp_path, model_id.replace("/", "_"))
+    subprocess.run(["optimum-cli", "export", "openvino", "--model", model_id, MODEL_PATH, "--task", task],
+                   capture_output=True,
+                   text=True)
+    return MODEL_PATH
 
 
-def setup_module():
-    for model_info in OV_RERANK_MODELS:
-        model_id = model_info[0]
-        task = model_info[1]
-        MODEL_PATH = Path(tmp_dir, model_id.replace("/", "_"))
-        subprocess.run(["optimum-cli", "export", "openvino", "--model", model_id, MODEL_PATH, "--task", task, "--trust-remote-code"],
-                       capture_output=True,
-                       text=True)
+def remove_artifacts(artifacts_path: Path, file_type="outputs"):
```

Comment:
Use `file_type` or remove from arguments
```suggestion
def remove_artifacts(artifacts_path: Path):
```